python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ntp.schema.yml

```diff
@@ -105,3 +105,33 @@ keys:
         description: List of trusted-keys as string ex. 10-12,15.
         convert_types:
           - int
+      serve:
+        description: Enable Serving NTP to clients
+        type: dict
+        keys:
+          all:
```

Comment:
Please remove the `all` level. The CLI does not contain `all` in every command, so we should probably have an `serve_all: <bool>` key under `serve` and rename the `enabled` key to `serve_all` under the VRFs. (Normally the key should be "all" but I think that would be less intuitive for the user).

Apologies if I told you to nest under `all`.